src/styles/blocks/main.scss

```diff
@@ -0,0 +1,14 @@
+.main {
+  background-color: $background-color;
+
+  &__content {
+    display: grid;
+    row-gap: 92px;
+    padding: 60px 0 120px;
+
+    @include onTablet {
+      padding-block: 120px;
+      row-gap: 120px;
+    }
+  }
```

Comment:
I don't think you should make single-column grid for main. Flex will be enough

Reply:
in this situation I choose grid, because for flex I should set flex-direction, it's one line of code more